magpie/ui/management/templates/add_group.mako

```diff
@@ -72,13 +73,13 @@
             </td>
         </tr>
         <tr>
-            <td>Terms and conditions:</td>
-            <td>
+            <td style="vertical-align: top">Terms and conditions:</td>
+            <td colspan="2">
                 <label>
-                    <input type="text" name="terms" value="${form_terms}" />
+                    <textarea rows="4" cols="50" style="min-width: 282px; min-height: 66px" name="terms">${form_terms}</textarea>
                 </label>
             </td>
-            <td>
+            <td style="vertical-align: top">
```

Comment:
Use a specific CSS class for `vertical-align` parts.
Use another CSS class (eg: `field-expandable`) to define anything specific to that feature.
There should be other field/table classes in the style file that affect the display of elements of this part. 

Employ `%` or `em` rather than `px` since they are different for each monitor. 
They seem specific for your display but could appear wrong for others.